crates/nu-command/src/strings/format/date.rs

```diff
@@ -61,6 +61,14 @@ impl Command for FormatDate {
                     Span::test_data(),
                 )),
             },
+            Example {
+                description: "Format a given date-time using the RFC3339 format.",
```

Comment:
```suggestion
                description: "Format a given date-time in RFC3339 standard.",
```

Reply:
Using "according to" as in the other comment. You can format according to or 'conform to' a standard.